greet.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable no-unused-vars */
+const greet = (name) => {
+  if (name) return "Hello, " + name + "!";
```

Comment:
I chose not to use the ternary operator because the string includes so many signs. Therefore, I thought using the if condition would make the code more readable and clear.🙂

Reply:
Okay